tests/integration/tei/test_tei_exporting.py

```diff
@@ -0,0 +1,19 @@
+"""Integration tests for the TEI exporter."""
+
+import pytest
+
+from inspirehep.modules.converttohal import tei
+from inspirehep.utils.record_getter import get_db_record
+
+SAMPLE = "tests/integration/tei/fixtures/sample.xml"
+
+
+def test_format_tei(app):
+    r = get_db_record('literature', 1407987)
```

Comment:
Ouch! It's a bit an empty record. Can you pick up from the demo records one with journals etc.?